nexus-repository-conan/src/test/java/org/sonatype/repository/conan/upgrade/ConanUpgrade_1_1_Test.groovy

```diff
@@ -0,0 +1,321 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype
+ * .com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License
+ * Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are
+ * trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.conan.upgrade
+
+import org.sonatype.goodies.testsupport.TestSupport
+import org.sonatype.nexus.orient.OClassNameBuilder
+import org.sonatype.nexus.orient.OIndexNameBuilder
+import org.sonatype.nexus.orient.testsupport.DatabaseInstanceRule
+import org.sonatype.repository.conan.internal.AssetKind
+import org.sonatype.repository.conan.internal.ConanFormat
+
+import com.google.common.collect.ImmutableMap
+import com.orientechnologies.orient.core.collate.OCaseInsensitiveCollate
+import com.orientechnologies.orient.core.db.record.OIdentifiable
+import com.orientechnologies.orient.core.index.OIndex
+import com.orientechnologies.orient.core.metadata.schema.OClass.INDEX_TYPE
+import com.orientechnologies.orient.core.metadata.schema.OSchema
+import com.orientechnologies.orient.core.metadata.schema.OType
+import com.orientechnologies.orient.core.record.impl.ODocument
+import org.hamcrest.Matchers
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.notNullValue
+
+class ConanUpgrade_1_1_Test
+    extends TestSupport
+{
+  static final String REPOSITORY_CLASS = new OClassNameBuilder()
+      .type("repository")
+      .build()
+
+  static final String I_REPOSITORY_REPOSITORY_NAME = new OIndexNameBuilder()
+      .type(REPOSITORY_CLASS)
+      .property(P_REPOSITORY_NAME)
+      .build()
+
+  static final String BUCKET_CLASS = new OClassNameBuilder()
+      .type("bucket")
+      .build()
+
+  static final String I_BUCKET_REPOSITORY_NAME = new OIndexNameBuilder()
+      .type(BUCKET_CLASS)
+      .property(P_REPOSITORY_NAME)
+      .build()
+
+  static final String ASSET_CLASS = new OClassNameBuilder()
+      .type("asset")
+      .build()
+
+  static final String I_ASSET_NAME = new OIndexNameBuilder()
+      .type(ASSET_CLASS)
+      .property(P_NAME)
+      .build()
+
+  static final List<String> ACTUAL_NAMES = Collections.unmodifiableList(Arrays.asList(
```

Comment:
just ImmutableList.of (... , ...)

Reply:
fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/95/commits/6bac95f207f3b84e6c3e28f23661ea59eb9c31fb